firestore-send-email/README.md

```diff
@@ -47,7 +47,7 @@ Usage of this extension also requires you to have SMTP credentials for mail deli
 
 * Cloud Functions location: Where do you want to deploy the functions created for this extension? You usually want a location close to your database. For help selecting a location, refer to the [location selection guide](https://firebase.google.com/docs/functions/locations).
 
-* SMTP connection URI: A URI representing an SMTP server that this extension can use to deliver email.
+* SMTP connection URI: A URI representing an SMTP server that this extension can use to deliver email. PLease note SMTP providers typically include ports `25`, `2525` and `587`. As port `25` is potentially blocked for security reasons, our recommendation would be to use the most recent `587` port for any integrations.
```

Comment:
I'd probably word it something like this:
Please note, port `25` is blocked by Google Cloud Platform, we recommend using port `587` for `smtp` connections. If you're using the `smtps` protocol, we recommend using port `465`.

Reply:
That sounds good to me, the first was just a rough draft to get the main details.

The above matches https://cloud.google.com/compute/docs/tutorials/sending-mail so happy to update.